components/ActionBar/Bookmark.js

```diff
@@ -90,7 +90,7 @@ const addMutation = gql`
       id
 #      userListItems {
```

Comment:
There's another issue maybe not related to, but at least triggered by `addDocumentToList`:

1. On localhost, navigate to the feed
2. Navigate to an article that is not bookmarked yet.
3. On the article's action bar, click on the bookmark icon to bookmark the article
4. Page return a 404 💥
5. Navigate back to the feed – the article was bookmarked successfully.

So the problem is the 404 which only occurs when bookmarking on the article (not when un-bookmarking).

Reply:
can repo, but maybe was related to https://github.com/orbiting/republik-frontend/pull/234/commits/e9721979d6bcac688faeb7fb22c7ef348fdbad61. Can you try to reproduce again